src/js/gist-it.js

```diff
@@ -28,10 +28,12 @@ $(function() {
 			window[callbackName] = undefined;
 
 			// If available, re-run prettify so that the new content is highlighted
-			if ("prettyPrint" in window) window.prettyPrint();
+			if ("prettyPrint" in window) {
+				window.prettyPrint();
+			}
 		};
 
-		var sc = document.createElement('script'); sc.type = 'text/javascript'; sc.async = true; sc.src = url;
+		var sc = document.createElement('script'); sc.src = url;
```

Comment:
Today browsers load asynchronously injected scripts by default so I think it's fine. Unless we want to support async for Firefox 2 and 3 in techdocs, but I believe that's not the case.